src/server/attestation/mod.rs

```diff
@@ -284,9 +287,15 @@ pub async fn attest(
 
                 let mut response = response_empty(&request);
                 response.set_status(ResponseType::Changed);
+
+                *state.clients_with_fts_access.lock().await += 1;
+                led::control(led::LedState::AttestationOk);
                 return Ok(response);
             }
-            Err(e) => return Err(e),
+            Err(e) => {
+                led::control(led::LedState::AttestationFailed);
```

Comment:
```suggestion
                led::control(led::LedState::AttestationFailed);
                led::control(led::LedState::TokenWaiting);
```

This fixes two issues:
- no blinking LEDs after failed attestation, obviously
- after failed attestation, followed by successful one, both red and green LEDs were lit instead of just green, due to `attestation_failed` not being cleared by https://github.com/fobnail/fobnail/pull/57/files#diff-7531f8c4a4c6d72ba8e487b65d4e4e4ed3eec12d5cf55b7cb439d436faf4b6faR122